images/node/system-container/service.template

```diff
@@ -21,4 +21,4 @@ WorkingDirectory=$DESTDIR
 RuntimeDirectory=${NAME}
 
 [Install]
-WantedBy=docker.service
+WantedBy=${DOCKER_SERVICE}
```

Comment:
We could also make this multi.user ?

Reply:
https://github.com/openshift/openshift-ansible/commit/93d48b330e500bd45a91c4dc7dc3d89fd000dfc4 provides justification for why this is as it is.